genienlp/paraphrase/scripts/transform_dataset.py

```diff
@@ -129,6 +131,9 @@ def main(args):
     if args.remove_duplicates and args.no_duplication_columns is None:
         raise ValueError('You should specify columns that define duplication')
 
+    if args.bootleg_output_file:
+        bootleg_output_file = open(args.bootleg_output_file, 'w')
```

Comment:
Given you load the whole input file in memory anyway, you should load the input file before you open the output file. That way, the same file can be both input and output.